Community/user_inventory/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 BlueCat Networks. All rights reserved.
```

Comment:
Please change date to 2019. Also in user_inventory_page.py.

Reply:
Dates fixed